src/main/webapp/js/instructorStudentListAjax.js

```diff
@@ -87,7 +116,7 @@ var seeMoreRequest = function(e) {
                 });
             }
         } else {
-            numStudents -= courseNumStudents;
+            // Failed to load
```

Comment:
Related to above, make this comment more specific? This branch is taken when the number of students is exceeded, so rather than failed to load, this is actually not loading the students at all?